src/utils/amazon.ts

```
@@ -0,0 +1,15 @@
+export function isValidAmazonWishlist(url: string): boolean {
+  try {
+    const u = new URL(url);
+    const host = u.hostname.toLowerCase();
+    if (!host.includes('amazon.')) return false;
```

ESLint

[error] Replace `'amazon.'` with `"amazon."`

(prettier/prettier)